scripts/docker/dev/Dockerfile

```diff
@@ -82,7 +82,7 @@ RUN mkdir -p $SPACK_ROOT \
 
 # TODO: delete when next spack version comes out
 # see https://github.com/spack/spack/pull/33128
-RUN if [ "$DOCKERFILE_DISTRO" == "centos" ] ; then \
+RUN if [ "$DOCKERFILE_DISTRO" = "centos" ] ; then \
```

Comment:
Does the tool/script language support both `=` and `==`?

Reply:
POSIX requires `=` between single brackets. I think the centos `/bin/sh` was bash (which allows it) but the ubuntu `/bin/sh` does not.